nbconvert/preprocessors/extractoutput.py

```diff
@@ -102,6 +106,13 @@ def preprocess_cell(self, cell, resources, cell_index):
                     out.metadata.setdefault('filenames', {})
                     out.metadata['filenames'][mime_type] = filename
 
+                    if filename in resources['outputs']:
+                        raise ValueError(
+                            "Your filename: {} appears more than once. "
```

Comment:
I think this message could be a bit clearer - imagine a user who has got a notebook from someone else and is trying to convert it. Even if someone wrote it themselves, they might accidentally duplicate a filename by copying a cell.

Let's mention that this is a filename associated with an *output*, which nbconvert is now saving to a separate file, and that the name has come from output metadata.